pkg/crc/cluster/cluster.go

```diff
@@ -23,18 +23,6 @@ import (
 // #nosec G101
 const vmPullSecretPath = "/var/lib/kubelet/config.json"
 
-func WaitForSSH(ctx context.Context, sshRunner *ssh.Runner) error {
-	checkSSHConnectivity := func() error {
```

Comment:
Why move it?

Reply:
@cfergeau because as my understanding this is more appropriate location for this function this is nothing to do with cluster side but waiting for SSH connection.

Reply:
Fine with me, but it's always useful to document the rationale in the commit log, both for easier reviews, and for posterity